tests/test_scripts.py

```diff
@@ -2777,6 +2775,52 @@ def test_trim_low_abund_trimtest_savetable():
                 'GGTTGACGGGGCTCAGGGGGCGGCTGACTCCGAGAGACAGCA'
 
 
+def test_counting_load_gzipped_bigcount():
+    import gzip
+    infile = utils.get_temp_filename('test_ct')
+    ct = khmer.new_counting_hash(10, 1e7, 4)
+    ct.set_use_bigcount(True)
+    for i in range(500):
+        ct.count('ATATATATAT')
+    ct.save(infile)
+    outfile = utils.get_temp_filename('test_ct.gz')
+    fin = open(infile, 'rb').read()
+    fout = gzip.open(outfile, 'wb')
+    fout.write(fin)
+    newct = khmer.new_counting_hash(10, 1e7, 4)
+    newct.load('test_ct.gz')
```

Comment:
You need to reference the temporary filename stored in the `outfile` variable on line 2786. Otherwise it will try to load `test_ct.gz` from wherever the current directory happens to be.